### PR TITLE
Use sass loader in the bike share example

### PR DESCRIPTION
**Pull request by doug-wade**
Yay! 🎉

### CODE REVIEW

File: packages/react-server-examples/bike-share/package.json [hunk 1]
```diff
@@ -7,35 +7,21 @@
...
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-runtime": "^6.6.1",
+    "babel-runtime": "^6.11.6",
     "react": "~0.14.2",
     "react-dom": "~0.14.2",
     "react-server": "^0.4.5",
     "react-server-cli": "^0.4.5",
+    "sass-loader": "^4.0.0",
```
Comment (doug-wade) [inline packages/react-server-examples/bike-share/package.json]: this feels like a bug in react-server-cli, tbqh

File: packages/react-server-examples/bike-share/package.json [hunk 2]
```diff
@@ -7,35 +7,21 @@
...
   },
   "devDependencies": {
     "babel-preset-react-server": "^0.4.4",
-    "eslint-config-xo-react": "^0.7.0",
-    "eslint-plugin-react": "^5.1.1",
-    "node-sass": "^3.7.0",
-    "nsp": "^2.3.3",
-    "xo": "^0.15.1"
-  },
-  "xo": {
```
Comment (doug-wade) [inline packages/react-server-examples/bike-share/package.json]: This is a react-server example, keep it as simple as possible.

File: packages/react-server-examples/bike-share/package.json [hunk 3]
```diff
@@ -7,35 +7,21 @@
   "author": "Doug Wade <doug.wade@redfin.com>",
   "scripts": {
     "clean": "rm -rf build __clientTemp",
-    "start": "npm run clean && npm run styles && react-server start",
-    "styles": "node-sass styles/index.scss build/styles/index.css && node-sass styles/network.scss build/styles/network.css",
-    "test": "xo && nsp check"
+    "start": "npm run clean && react-server start --hot",
```
Comment (gigabo) [inline packages/react-server-examples/bike-share/package.json]: Wow, so much cleaner!

Should we move the `hot` into a `.reactserverrc`?

File: packages/react-server-examples/bike-share/package.json [hunk 4]
```diff
@@ -7,35 +7,21 @@
...
+    "test": "nsp check"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-runtime": "^6.6.1",
+    "babel-runtime": "^6.11.6",
```
Comment (gigabo) [inline packages/react-server-examples/bike-share/package.json]: This _shouldn't_ even be necessary?

Should only need the packages that are installed by [`react-server init`](https://github.com/redfin/react-server/blob/master/packages/react-server-cli/src/commands/init.js#L8-L16).

File: packages/react-server-examples/bike-share/package.json (hunk 1 above)
Comment (gigabo) [inline packages/react-server-examples/bike-share/package.json]: Yeah, this shouldn't be necessary either. 😕

Comment (gigabo) [inline packages/react-server-examples/bike-share/package.json]: Have you run `npm i` in `packages/react-server-cli` since the dep was added there?

**Comment by gigabo** [comment]
This LGTM, though I'm confused by the babel/sass-loader deps.

File: packages/react-server-examples/bike-share/package.json (hunk 3 above)
Comment (doug-wade) [inline packages/react-server-examples/bike-share/package.json]: It seems like a lot of overhead to add a whole new file to eliminate one flag...

**Comment by doug-wade** [comment]
@gigabo pulled the sass and babel deps.  I had to run a relative install (`npm i ../react-server-cli`) to get it to pass locally -- `lerna bootstrap` didn't seem to fix it -- and I'm not sure what that means about whether we should merge this.

**Comment by gigabo** [comment]
Yeah, `lerna bootstrap` can't help us with the examples ([yet](https://github.com/lerna/lerna/issues/126)) because they're not direct children of `packages/`.  But... the install from `^0.4.5` should work. 😕

**Pull request merged**